src/common/config.go

```diff
@@ -64,6 +64,9 @@ type Config struct {
 	// which OCI implementation to use for the docker sandbox (e.g., runc or runsc)
```

Comment:
As Config gets more complicated, some fields are only relevant to certain implementations.  E.g., Docker_runtime and Docker_base_image are only used when we use Docker containers.  It feels like we should isolate implementation-specific fields into sub structs.